src/main/scala/org/globalnames/parser/ParserClean.scala

```diff
@@ -310,8 +310,8 @@ class ParserClean extends SimpleParser {
 
   val unknownAuthor: Rule1[Author] = rule {
     capture("?" |
-            (("auct." | "auct" | "anon." | "anon" | "ht." | "ht" | "hort." |
-              "hort") ~ &(spaceChars | EOI))) ~>
+            (("auct" | "anon" | "ht" | "hort") ~ '.'.? ~
```

Comment:
hort. anon. auct. mean different types of unknown, so it is better to keep them in normalized version and in authorship, in semantic explanation I chose to use 'unknown' before and noone complained.